csharp/extractor/Semmle.Extraction.CSharp/Entities/Keyword.cs

```diff
@@ -0,0 +1,20 @@
+internal static class Keyword
+{
+    public static readonly string Abstract = "abstract";
```

Comment:
Hehe, that is interesting. I have just been so used to using `static readonly` instead of `const` as it has a semantic difference in case all the code is not re-compiled as `const` turns the value into a compile time constant and it is then been inserted in all the places where it is referenced. This can cause a problem if a const value has a tendency to change (which will not happen here) and if not all the projects that reference the values are rebuild.
Since that is not likely to happen here, I will change it to const.